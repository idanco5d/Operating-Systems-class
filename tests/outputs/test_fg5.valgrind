==21530== Memcheck, a memory error detector
==21530== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==21530== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==21530== Command: /mnt/c/Users/illus/os/hw/1/smash
==21530== Parent PID: 21524
==21530== 
==21530== 
==21530== FILE DESCRIPTORS: 4 open (3 std) at exit.
==21530== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_fg5.valgrind
==21530==    <inherited from parent>
==21530== 
==21530== 
==21530== HEAP SUMMARY:
==21530==     in use at exit: 0 bytes in 0 blocks
==21530==   total heap usage: 86 allocs, 86 frees, 82,211 bytes allocated
==21530== 
==21530== All heap blocks were freed -- no leaks are possible
==21530== 
==21530== For lists of detected and suppressed errors, rerun with: -s
==21530== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
