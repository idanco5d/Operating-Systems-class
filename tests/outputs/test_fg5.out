smash> submitted by Eran
smash> smash> smash> smash> sleep 200& : 9119
smash: got ctrl-Z
smash: process 9119 was stopped
smash> [1] sleep 100& : 9118 2 secs
[2] sleep 200& : 9119 2 secs (stopped)
[3] sleep 300& : 9120 2 secs
smash> 